src/components/programmingCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import "./programmingCard.scss"
+
+export default function ProgrammingCard({ title, summary, id }) {
+  const href = `https://www.artsmatter.org/programs/${id}`
```

Comment:
Can you make this a relative path so that it will work on our local/staging environments as well? Also, to me using the id (which is a random string) as the page slug seems like a bad idea to me. I'm going to discuss this with Arts Matter.

I would rather it look like `https://artsmatter.org/programs/program-name`

For now, let's just make this path relative to the current domain and we can swing back around to the slug later.

Reply:
Yeah, I was curious if it would be better to loop through the programs and display them, that would be a good improvement to what I have in regards to art-matter being able to update that header area. It would be great to go over that later with Tim. I'm mainly curious about how to order them correctly since they're not in any particular order in contentful, as well as how to know if a card is an expandable version or not and what programs will need to go in an expandable card. Some things to discuss later. And of course, I will switch those to be relative links. I'll be interested to dig into how to create these dynamic routes using gatsby or if we will just add them as hardcoded pages.